docs/book/storage/adapter.md

```diff
@@ -561,6 +561,50 @@ Name | Data Type | Default Value | Description
 ---- | --------- | ------------- | -----------
 `namespace_separator` | `string` |  ":" | A separator for the namespace and prefix.
 
+## The BlackHole Adapter
+
+`Laminas\Cache\Storage\Adapter\BlackHole` **does not** store any cache items.
```

Comment:
A description of the possible use should be added here. Like: "This adapter is mostly useful for tests."